cypress/e2e/data-import/filter-and-delete-import-logs-on-view-all.cy.js

```diff
@@ -71,8 +71,7 @@ describe('ui-data-import: A user can filter and delete import logs from the "Vie
     Users.deleteViaApi(secondUser.userId);
     // TODO delete all created instances and holdings
   });
-  // removed tag smoke because test runs infinite
-  it('C358136 A user can filter and delete import logs from the "View all" page (folijet)', { tags: [DevTeams.folijet] }, () => {
+  it('C358136 A user can filter and delete import logs from the "View all" page (folijet)', { tags: [DevTeams.folijet, DevTeams.folijet] }, () => {
```

Comment:
line 2
'TestTypes' is defined but never used. Allowed unused vars must match /React/u

Reply:
Done